ts/Dashboards/Plugins/DataGridSyncHandlers.ts

```diff
@@ -210,13 +210,72 @@ const configs: {
                 this.on('setConnector', (): void => unregisterCursorListeners());
                 this.on('afterSetConnector', (): void => registerCursorListeners());
             }
+        },
+        visibilityHandler: function(this: DataGridComponent): void {
+            const component = this;
+            const { board } = this;
```

Comment:
Would be better to use component here.

`const { board } = component;`

It will be more clearful.